services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/message/DataAccessSubmissionStatusMessageBuilderFactory.java

```diff
@@ -25,8 +31,11 @@ public BroadcastMessageBuilder createMessageBuilder(String objectId, ChangeType
 				|| submission.getState().equals(DataAccessSubmissionState.APPROVED),
 				"DataAccessSubmissionState not supported: "+submission.getState());
 		boolean isRejected = submission.getState().equals(DataAccessSubmissionState.REJECTED);
+		List<RestrictableObjectDescriptor> subjects = accessRequirementDao.getSubjects(Long.parseLong(submission.getAccessRequirementId()));
```

Comment:
Calling get list and only using the first is asking the DB to send more data than is needed.  Can we add a method to get the single object?